label_studio/templates/tasks.html

```diff
@@ -1,81 +1,102 @@
+          {% if task_ids|length > 0 %}
+            <a class="ui button positive" href="/">Start Labeling</a>
+            <a id="clear-tasks-button" class="ui button red">Delete All Tasks</a>
+          {% else %}
+            <a class="ui button positive" href="/import">Import Tasks</a>
+          {% endif %}
+          <br/><br/>
+        </div>
 
+      </div>
 
-    <!-- Table -->
-    <table>
-      <tr>
-        <th>Task ID</th>
-        <th>Completed at</th>
-        <th>Completion</th>
-      </tr>
 
-      {% for id in task_ids %}
+      <!-- Table -->
+      <table>
         <tr>
-          <td class="text-center">
-            <a href="/?task_id={{ id }}">
-              <i class="fas fa-eye eye show-completion"></i>
-            </a>
-            &nbsp;
-            {{ id }}
-          </td>
-          <td>{{ completed_at[id] }}</td>
-          <td class="text-center">
-            {% if id in completions %}
-              <i data-task-id="{{ id }}" class="fas fa-trash-alt trash remove-completion"></i>
-            {% endif %}
-          </td>
+          <th>Task ID</th>
+          <th>Completed at</th>
+          <th>Completion</th>
         </tr>
-      {% endfor %}
-    </table>
-    <br/>
+
+        {% for id in task_ids %}
+          <tr>
+            <td class="text-center">
+              <a href="/?task_id={{ id }}">
+                <i class="fas fa-eye eye show-completion"></i>
+              </a>
+              &nbsp;
+              {{ id }}
+            </td>
+            <td>{{ completed_at[id] }}</td>
+            <td class="text-center">
+              {% if id in completions %}
+                <i data-task-id="{{ id }}" class="fas fa-trash-alt trash remove-completion"></i>
+              {% endif %}
+            </td>
+          </tr>
+        {% endfor %}
+      </table>
+      <br/>
   </div>
-</div>
+  <br/><br/>
+  </div>
+
 
+  <script>
+    document.querySelectorAll(".remove-completion").forEach(function (test) {
+      test.addEventListener("click", function (event) {
+        var id = event.target.dataset.taskId;
+        var request = new XMLHttpRequest();
+        request.open("DELETE", "/api/tasks/" + id + "/completions/" + id + "/", true);
+        request.onload = function () {
+          window.location.reload();
+        };
+        request.send(null);
+      })
+    });
 
-<script>
-  document.querySelectorAll(".remove-completion").forEach(function (test) {
-    test.addEventListener("click", function (event) {
-      var id = event.target.dataset.taskId;
-      var request = new XMLHttpRequest();
-      request.open("DELETE", "/api/tasks/" + id + "/completions/" + id + "/", true);
-      request.onload = function () {
-        window.location.reload();
+    document.getElementById("clear-tasks-button").onclick = function (e) {
+      const sure = 'Are you sure?';
+      if ($(e.target).text() !== sure) {
+        $(e.target).text(sure);
```

Comment:
It seems a bit strange to confirm action through changing button name, at least this screen without context looks weird ;)
![image](https://user-images.githubusercontent.com/6087484/72669191-fd956b00-3a3f-11ea-8625-52ca1da690a4.png)
Can we try modal window here?